### PR TITLE
Create newsletter-2024-06.md

### PR DESCRIPTION
**Pull request by Davsarper**
copy over from hackmd to make crm task easier

### CODE REVIEW

**Review by gedankenstuecke — APPROVED** [review]

File: 00-project-documentation/communication/newsletters/newsletter-2024-06.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Hi from AutSPACEs! 
+
+We're no longer using TinyLetter to send out our newsletter. Instead, we're now managing our mailing list using Turing's CRM and the email address AutSPACEs@turing.ac.uk. To share our newsletter with others who might be interested please ask them to email us at AutSPACEs@turing.ac.uk and we will add them to the mailing list. You can also message us using this address if you want to be removed from the mailing list. 
+
+Our next meet-up session is coming up next Thursday, 4th July. In this session we'll be continuing the work we started last month on writing a chapter on co-creation for The Turing Way as well as on an AutSPACEs glossary. Meet-up sessions are open to anyone interested in taking part or learning more. To join just follow the zoom link. If you're new to AutSPACEs, it's also a space where we can onboard you onto the project.  
+
+**Date:** 
+
+Thursday 4th July
```
Comment (Davsarper) [inline 00-project-documentation/communication/newsletters/newsletter-2024-06.md:12]:
```suggestion
**Date:** Thursday 4th July 
```

File: 00-project-documentation/communication/newsletters/newsletter-2024-06.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+**Date:** 
+
+Thursday 4th July 
+
+**Time:** 12:00-1:30pm
+
+**Location:**     
+
+   Online via. Zoom: https://turing-uk.zoom.us/j/523235071
```
Comment (Davsarper) [inline 00-project-documentation/communication/newsletters/newsletter-2024-06.md:18]:
```suggestion
**Location:**  Online via Zoom: https://turing-uk.zoom.us/j/523235071
```

File: 00-project-documentation/communication/newsletters/newsletter-2024-06.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+   
+**Agenda and notes:**
+
+https://docs.google.com/document/d/1YkVZYQr7T2QnDxfX61EWU-tr_Pjwl0b8ixWbZ27CiE8/edit?usp=sharing
```
Comment (Davsarper) [inline 00-project-documentation/communication/newsletters/newsletter-2024-06.md:22]:
```suggestion
**Agenda and notes:** https://docs.google.com/document/d/1YkVZYQr7T2QnDxfX61EWU-tr_Pjwl0b8ixWbZ27CiE8/edit?usp=sharing
```

Comment (Davsarper) [inline 00-project-documentation/communication/newsletters/newsletter-2024-06.md:22]: Suggest details are always in one line each (at least in CRM)

File: 00-project-documentation/communication/newsletters/newsletter-2024-06.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+You can also join us at the next meet-up session on Thursday July 4th or email [Georgia](mailto:gaitkenhead@turing.ac.uk) to learn more and have an introductory dicussion. 
+
+## Glossary 
+
+We're putting together a glossary of terms which is also a language guide for words and terms and how best to use them. If you'd like to contribute to the glossary you can do so by following this link and adding any new words, definitions, or comments: 
+
+https://docs.google.com/document/d/1zqMm7Ne1rAPjC1drdqNZ_x2_m5kfRmdbAS3x4gQFBXI/edit?usp=sharing. 
+
+As with the chapter for The Turing Way, you can also join us at the next meet-up session on Thursday July 4th or [Georgia](mailto:gaitkenhead@turing.ac.uk) to learn more and have an introductory dicussion.
```
Comment (Davsarper) [inline 00-project-documentation/communication/newsletters/newsletter-2024-06.md:40]:
```suggestion
As with the chapter for The Turing Way, you can also join us at the next meet-up session on Thursday July 4th or email [Georgia](mailto:gaitkenhead@turing.ac.uk) to learn more and have an introductory dicussion.
```

File: 00-project-documentation/communication/newsletters/newsletter-2024-06.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+## Spotlight Interview - Anne Lee Steele
+
+![image](https://hackmd.io/_uploads/SJRc50cUC.png)
+
+Today's spotlight interview is with Anne Lee Steele. She is the community manager of The Turing Way and some of you have met her at a recent co-working call. We're really delighted to have her support bringing together communities and sharing our learning from AutSPACEs. 
+
+>### Tell us a bit about yourself
+>
+>I'm Anne, the Community Manager of The Turing Way project! Through our open access resources, we aim to "reproducibility too easy not to do" and help make data-driven research more open, collaborative, inclusive and accessible. I'm an anthropologist by training, which means that I'm always interested in how cultures might evolve and change – which is what our project is all about! In my free time, I also make multi-sensory digital art, which comes from a desire to learn about how we can engage about in complex topics through other senses besides our sight  :slightly_smiling_face:
```
Comment (Davsarper) [inline 00-project-documentation/communication/newsletters/newsletter-2024-06.md:50]:
```suggestion
>I'm Anne, the Community Manager of The Turing Way project! Through our open access resources, we aim to make "reproducibility too easy not to do" and help make data-driven research more open, collaborative, inclusive and accessible. I'm an anthropologist by training, which means that I'm always interested in how cultures might evolve and change – which is what our project is all about! In my free time, I also make multi-sensory digital art, which comes from a desire to learn about how we can engage about in complex topics through other senses besides our sight.  :slightly_smiling_face:
```

File: 00-project-documentation/communication/newsletters/newsletter-2024-06.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+>
+>### What are you doing with AutSPACEs?
+>
+>A few years ago, Georgia and I hosted a fireside chat about citizen science, which really highlighted the overlaps between our communities. We're finding more overlaps between our ways of working and overlapping projects, and I'm keen to support the new chapter on co-creation. :slightly_smiling_face:
+>
+>### What's your tip for how to get involved/contribute?
+>
+>Get started with something small – fix a typo on a page or join a community call to say hello! And don't be afraid to be a lurker for a while before joining – it's an important way of learning about community culture before joining :slightly_smiling_face:
```
Comment (Davsarper) [inline 00-project-documentation/communication/newsletters/newsletter-2024-06.md:58]:
```suggestion
>Get started with something small – fix a typo on a page or join a community call to say hello! And don't be afraid to be a lurker for a while before joining – it's an important way of learning about community culture before joining. :slightly_smiling_face:
```

**Review by Davsarper — COMMENTED** [review]
suggested changes from close read, couple of missing words basically

**Pull request merged**